subiquity/models/filesystem.py

```diff
@@ -665,13 +663,13 @@ def annotations(self):
         return []
 
     def desc(self):
+        if self.multipath:
+            return "multipath"
```

Comment:
Would it make sense to include either the path or the mpath name or both?

Reply:
This is a "type" descriptor column, in short line form. For other things it's "Software RAID", "LVM", "local disk". The full line looks like this "$WWN        multipath" with the second line describing quickly what's on that drive.

mapth[a-z] are random, and WWN is the important bit, which is listed. So "multipath" is here to destinguish from "local disk" which isn't a multipathed one.

Reply:
Other desc() are "local disk", "software RAID 1", "LVM volume group", "LVM logical volume"